ceph_deploy/mon.py

```diff
@@ -58,7 +58,7 @@ def catch_mon_errors(conn, logger, hostname, cfg, args):
     and warn apropriately about it.
     """
     monmap = mon_status_check(conn, logger, hostname, args).get('monmap', {})
-    mon_initial_members = cfg.safe_get('global', 'mon_initial_members')
+    mon_initial_members = get_mon_initial_members(args, error_on_empty=True, _cfg=cfg)
```

Comment:
I don't think this one should have error_on_empty=True.  The follow-on logic is counting on mon_initial_members being `None` if there are no members.